File: ios/TMapShowManager.m
//
//  TMapShowManager.m
//  EVChargerApp
//
//  Created by Joseph Kim on 2021/03/14.
//

#import <Foundation/Foundation.h>

#import "React/RCTViewManager.h"
@interface RCT_EXTERN_MODULE(TMapShowManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(zoom, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(clatitude, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(clongitude, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(markerdata, NSArray)
RCT_EXPORT_VIEW_PROPERTY(dlatitude, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(dlongitude, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(routedata, NSArray)
@end

File: ios/TMapShowManager.swift
//
//  TMapShowManager.swift
//  EVChargerApp
//
//  Created by Joseph Kim on 2021/03/14.
//

import Foundation

import TMapSDK

@objc(TMapShowManager)
class TMapShowManager: RCTViewManager {
  override func view() -> UIView! {
    return TMapShow()
  }
  
  override static func requiresMainQueueSetup() -> Bool {
      return true
}
}
